cluster/examples/kubernetes/rook-operator.yaml

```diff
@@ -79,27 +85,27 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: rook-operator
-  namespace: default
+  namespace: rook-system
 ---
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: rook-operator
-  namespace: default
+  namespace: rook-system
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: rook-operator
 subjects:
 - kind: ServiceAccount
   name: rook-operator
-  namespace: default
+  namespace: rook-system
 ---
 apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
   name: rook-operator
-  namespace: default
+  namespace: rook-system
```

Comment:
let's put this change in the PendingReleaseNotes.md. While it's not required to put the operator in the rook-system namespace, we should document the change

Reply:
Where is this document? I wasnt aware there was PendingReleaseNotes.md doc.

Reply:
Ahh i see it thanks. Thinking about more about it, i dont think this is a breaking change. `rook-system` just happen to be the namespace that these example manifests are using. The users can change this namespace and no where in the code am i hardcoding or defaulting to `rook-system`

Reply:
The release notes have notable features or changes in addition to the breaking changes, so this seems like a good thing to mention.

Reply:
ill mention it.